PPI/manual_work/Jul_2023/manual_tables.sql

```diff
@@ -0,0 +1,383 @@
+-- concept_manual backup
```

Comment:
Backups are no longer needed after the introduction of common manual tables - to be removed

Reply:
done